src/odb/include/odb/db.h

```diff
@@ -2815,7 +2815,8 @@ class dbInst : public dbObject
   /// There are seven methods used to get/set the placement.
   ///
   ///     getOrigin           - Get the origin of this instance (Where the
-  ///     master is setOrigin           - Set the origin of this instance
+  ///                           master is 
```

Comment:
You have trailing whitespace after is.  I don't think `(Where the master is` adds anything useful and I would just remove it (or at a minimum add a closing paren).

Reply:
I agree with removing `(Where the master is`.